.circleci/config.yml

```diff
@@ -1,4 +1,68 @@
-version: 2
+version: 2.1
+
+orbs:
+  lcdb-wf:
+    commands:
+      deploy-clone:
+        parameters:
+          flavor:
+            default: "full"
+            type: string
+        steps:
+          - run:
+              name: Deploy to a new directory, cloning to a temp dir
+              command: |
+                source /opt/mambaforge/etc/profile.d/conda.sh
+                conda activate $LCDBWF_ENV
+                conda info --envs
+                conda config --show
+
+                # Copy the deploy script to a different location to simulate the
+                # suggested deployment method of downloading just the script.
+                cp deploy.py /tmp/deploy.py
+                cd /tmp/
+
+                # Deploy to the new directory, so we are testing the real-world case of post-deployment.
+                # Note that $DEPLOY is set in the "set-paths" step configured above.
+                python deploy.py --flavor <<parameters.flavor>> --dest $DEPLOY --branch $CIRCLE_BRANCH --clone
+      get-data:
+        parameters:
+          kind:
+            default: "all"
+            type: enum
+            enum: ["all", "rnaseq", "chipseq"]
+          workflow:
+            type: string
+        steps:
+          - run:
+              name: Download example data
+              command: |
+                set -x
```

Comment:
The `tree` stuff left over from my debugging on the last PR, I could go either way with keeping it. I guess it doesn't _cost_ anything.